components/infoColumns/index.tsx

```diff
@@ -60,31 +59,32 @@ const CardContainer = styled.div<{
   ${breakpoints("margin", "", [{ 1400: "1rem 0" }])};
 `;
 const Title = styled(SubTitle)`
-  font-size: 4rem;
-  ${breakpoints("font-size", "", [{ 1000: "2.5rem" }])};
+  font-size: 6rem;
+  ${breakpoints("font-size", "", [{ 1000: "2.25rem" }])};
   width: 100px;
   ${breakpoints("width", "", [{ 1000: "100%" }])};
-  margin: 0;
+  margin: 0 0 0 0;
```

Comment:
Lol why change 0 into 0 0 0 0

Reply:
I think I added something and changed my mind completely and changed everything to 0s

Reply:
000oooo00000oooooOOOooOOOOoOo